pkg/kopia/command/storage/filesystem_args.go

```diff
@@ -0,0 +1,19 @@
+package storage
+
+import (
+	"github.com/kanisterio/kanister/pkg/logsafe"
+)
+
+const (
+	filesystemSubCommand = "filesystem"
+	pathFlag             = "--path"
+	// DefaultFSMountPath is the mount path for the file store PVC on Kopia API server
+	DefaultFSMountPath = "/mnt/data"
+)
+
+func filesystemArgs(location map[string]string, artifactPrefix string) logsafe.Cmd {
```

Comment:
Let's use a consistent function naming format. Other arg functions have `kopia*` prefix but not this one.